CHANGELOG.md

```diff
@@ -1470,7 +1470,7 @@ This file contains the changelog starting from v7.15.0.
 * `babel-parser`
   * [#13982](https://github.com/babel/babel/pull/13982) Extend `hasPlugin` to accept plugin-configuration array pairs ([@js-choi](https://github.com/js-choi))
 * `babel-helpers`
-  * [#13996](https://github.com/babel/babel/pull/13996) Don't use ESM for `require`d files in `@babel/helpers` tests ([@nicolo-ribaudo](https://github.com/nicolo-ribaudo))
+  * [#13996](https://github.com/babel/babel/pull/13996) Don't use ESM for `required` files in `@babel/helpers` tests ([@nicolo-ribaudo](https://github.com/nicolo-ribaudo))
```

Comment:
This one is wrong, I'm pretty sure. I don't know if there's a standard accepted way to apply the present perfect tense to a quoted word, but this is about files imported via `require`.

Reply:
I prefer `require`d: The full verb is "required", and the `require` part of that is in backticks because it's code (i.e. the name of the `require` function.
```suggestion
  * [#13996](https://github.com/babel/babel/pull/13996) Don't use ESM for `require`d files in `@babel/helpers` tests ([@nicolo-ribaudo](https://github.com/nicolo-ribaudo))
```

Reply:
Yes, You guys are right....my bad